modules/navier_stokes/src/materials/INSFVEnthalpyMaterial.C

```diff
@@ -31,11 +37,22 @@ INSFVEnthalpyMaterial::INSFVEnthalpyMaterial(const InputParameters & parameters)
     _temperature(getFunctor<ADReal>("temperature")),
     _cp(getFunctor<ADReal>(NS::cp))
 {
-  addFunctorProperty<ADReal>("rho_cp_temp",
-                             [this](const auto & r, const auto & t) -> ADReal
-                             { return _rho(r, t) * _cp(r, t) * _temperature(r, t); });
+  const auto & rho_h =
+      addFunctorProperty<ADReal>(NS::enthalpy_density,
+                                 [this](const auto & r, const auto & t)
+                                 { return _rho(r, t) * _cp(r, t) * _temperature(r, t); });
+
+  const auto & h = addFunctorProperty<ADReal>(NS::specific_enthalpy,
+                                              [this](const auto & r, const auto & t)
+                                              { return _cp(r, t) * _temperature(r, t); });
```

Comment:
I amended the doc string for `cp` to say it should be a constant. I do not know how to integrate some arbitrary expression for `_cp` here in this functor. In the Aria material since I have an analytic expression for cp, I implement the analytic integral for cp dtau for h

Reply:
oh yeah my bad. 

We should just query _fp directly for h instread of _cp, we should not try to integrate ourselves

Reply:
doc string update is fine